jpype/_jpackage.py

```diff
@@ -36,8 +36,6 @@ def __getattribute__(self, n):
                import warnings
                warnings.warn("JVM not started yet, can not inspect JPackage contents")
                return n
-            if not _jpype.isThreadAttachedToJVM():

```

Comment:
Threads are attached automatically at module level.  So this attach when package is created is unnecessary.